dhis-2/dhis-test-integration/src/test/java/org/hisp/dhis/tracker/export/event/AclEventExporterTest.java

```diff
@@ -384,7 +414,8 @@ void shouldFailWhenNoOuModeSpecifiedAndUserHasNoAccessToOrgUnit() {
   void shouldReturnAccessibleOrgUnitEventsWhenNoOrgUnitSpecifiedNorOuModeSpecified()
       throws ForbiddenException, BadRequestException {
     injectSecurityContext(userService.getUser("FIgVWzUCkpw"));
-    EventOperationParams params = EventOperationParams.builder().programUid("pcxIanBWlSY").build();
+    EventOperationParams params =
+        EventOperationParams.builder().programUid("pcxIanBWlSY").orgUnitMode(ACCESSIBLE).build();
```

Comment:
The test is called

`shouldFailWhenNoOuModeSpecifiedAndUserHasNoAccessToOrgUnit`

and should probably be called

`shouldFailWhenUserHasNoAccessToOrgUnit` or `shouldFailWhenUserHasNoAccessToOrgUnitGivenOrgUnitModeAccessible`

Reply:
Same as above, this test will be deleted, it has no value anymore.